Gemfile

```diff
@@ -87,6 +87,7 @@ group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
   gem "letter_opener"
+  gem "brakeman"
```

Comment:
should that be ``gem "brakeman", :require => false``
so bundler won't require it when start rails.